pydsa/tests/test_queue.py

```diff
@@ -0,0 +1,16 @@
+from pydsa.queue import Queue
+
+
+def test_queue():
+    def __init__self(self, size):
```

Comment:
This is not how the tests for the Queue should be written. You should first create a Queue object like this: `q = Queue()` and then do operations on it.